packages/destination-actions/src/destinations/marketo-static-lists/functions.ts

```diff
@@ -152,3 +168,113 @@ function parseErrorResponse(response: MarketoResponse) {
   }
   throw new IntegrationError(response.errors[0].message, 'INVALID_RESPONSE', 400)
 }
+
+export async function getAccessToken(request: RequestClient, settings: Settings) {
+  const api_endpoint = formatEndpoint(settings.api_endpoint)
+  const res = await request<RefreshTokenResponse>(`${api_endpoint}/${OAUTH_ENDPOINT}`, {
+    method: 'POST',
+    body: new URLSearchParams({
+      client_id: settings.client_id,
+      client_secret: settings.client_secret,
+      grant_type: 'client_credentials'
+    })
+  })
+
+  return res.data.access_token
+}
+
+export async function getList(request: RequestClient, settings: Settings, id: string) {
+  const accessToken = await getAccessToken(request, settings)
+  const endpoint = formatEndpoint(settings.api_endpoint)
+
+  const getListUrl = endpoint + GET_LIST_ENDPOINT.replace('listId', id)
+
+  const getListResponse = await request<MarketoListResponse>(getListUrl, {
+    method: 'GET',
+    headers: {
+      authorization: `Bearer ${accessToken}`
+    }
+  })
+
+  if (!getListResponse.data.success && getListResponse.data.errors) {
+    return {
+      error: {
+        message: getListResponse.data.errors[0].message,
+        code: 'LIST_ID_VERIFICATION_FAILURE'
+      }
+    }
+  }
+
+  return {
+    successMessage: `Using existing list ${id}.`,
```

Comment:
Maybe return the name here as well? Just a suggestion
```suggestion
    successMessage: `Using list '${getListResponse.data.result[0].name}' (id: ${id})`,
```